sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/LnStatus.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * // SPDX-FileCopyrightText: 2022 RTE FRANCE
+ * //
+ * // SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.lfenergy.compas.sct.commons.util;
+
+import lombok.experimental.UtilityClass;
+
+/**
+ * A representation of a specific object <em><b>TDAI</b></em> name that have attribute name STVal.
+ *
+ */
+@UtilityClass
```

Comment:
UtilityClass is experimental , and I remember that you don't like experimental stuff (Java 17 Enum pattern matching)  :smile:  
If this really is constant class, we can add a constructor ?  
```java
public LDeviceStatus() {
        new RuntimeException("should not be instanciated");
    }
```

Reply:
No problem. I delombok (it's the same code I you suggested). But I do that on every UtilityClass in Compas.